src/features/feed/img/imgLinks.js

```diff
@@ -6,3 +6,7 @@ export const foodBannerImage =
 
 export const freshBannerImage =
     'https://res.cloudinary.com/deqt3envc/image/upload/v1556741830/samples/Bukka-Images/banner-fresh.png';
+
+export const MartBannerImage =
+    'https://res.cloudinary.com/deqt3envc/image/upload/v1556741830/samples/Bukka-Images/banner-mart.png';
+
```

Comment:
Yeah, I almost ignored this. Where is this image coming from? have you uploaded to Cloudinary? Seems you just change the last name of the previous URL from `/banner-fresh.png` to `/banner-mart.png` :)